Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -270,6 +271,30 @@ LayoutUnit GridTrackSizingAlgorithm::initialGrowthLimit(const GridTrackSize& tra
     return infinity;
 }
 
+void GridTrackSizingAlgorithm::sizeTrackToFitNonSpanningItem(const GridSpan& span, LayoutUnit& singleTrackMinContentForChild, LayoutUnit& singleTrackMaxContentForChild, LayoutUnit& singleTrackMinSizeForChild, GridTrack& track)
```

Comment:
Do we need to pass these `LayoutUnit`s by reference? They don't seem to be out parameters and I think we generally pass them by value since they are small